apps/console/src/features/core/components/header.tsx

```diff
@@ -184,13 +184,14 @@ export const Header: FunctionComponent<HeaderPropsInterface> = (
      * Get the header extensions.
      */
     useEffect(() => {
-        commonConfig?.header?.getHeaderExtensions().then((response: HeaderExtension[]) => {
-            if (isPrivilegedUser) {
-                response.pop();
-            }
-            setHeaderExtensions(response);
-        });
-    }, []);
+        commonConfig?.header?.getHeaderExtensions(tenantDomain, associatedTenants)
+            .then((response: HeaderExtension[]) => {
+                if (isPrivilegedUser) {
+                    response.pop();
+                }
+                setHeaderExtensions(response);
+            });
+    }, [  tenantDomain, associatedTenants ]);
```

Comment:
Remove the extra space here.
```suggestion
    }, [ tenantDomain, associatedTenants ]);
```

Reply:
Fixed with 8edc053